androidsdk/src/main/java/com/wootric/androidsdk/Wootric.java

```diff
@@ -142,6 +142,22 @@ public void setCustomThankYou(WootricCustomThankYou customThankYou) {
         settings.setCustomThankYou(customThankYou);
     }
 
+    public void setSurveyColor(int surveyColor) {
```

Comment:
Is there a place where I can see which colors can I use? I'm not familiar with R so forgive me if the question it's too simple